instrumentation/rails/README.md

```diff
@@ -30,6 +30,18 @@ OpenTelemetry::SDK.configure do |c|
 end
 ```
 
+
+### Configuration options
+
+The `http.route` attribute is disabled by default because we use [.recognize](https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/journey/router.rb#L65)
```

Comment:
Instead of pointing to the main branch which may lose context as it changes, lets just use the current latest tag.
```suggestion
The `http.route` attribute is disabled by default because we use [.recognize](https://github.com/rails/rails/blob/v6.1.3/actionpack/lib/action_dispatch/journey/router.rb#L65)
```